src/components/Alert.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { createStyles, makeStyles } from "@material-ui/core/styles";
+import { COLORS } from "../utils/colors";
+import CheckCircleIcon from "@material-ui/icons/CheckCircle";
+import CancelIcon from "@material-ui/icons/Cancel";
+
+interface AlertProps {
+  severity: "error" | "success";
+}
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    container: {
+      display: "flex",
+      flexDirection: "row",
+      padding: 16,
+      borderRadius: 4,
+      alignItems: "center",
+    },
+    error: {
+      color: COLORS.rose["600"],
```

Comment:
We should not depend on an external file to retrieve the variables of the theme, but instead, use the parameters variables from the `makeStyles` function, an example here: https://material-ui.com/styles/api/#examples-2

Reply:
We could do it for some of the colors used. Material UI doesn't allow to add new colors to your pallet, only allow to configure a limit pallet of colors. https://stackoverflow.com/questions/53867755/setting-additional-colors-in-theme-material-ui-react